### DISCUSSION TITLE
推薦書單「蛤蟆先生去看心理師」

### DISCUSSION THREAD

**Discussion by Brownnnn78** [1101-GS4719-C/Python-Community-GS4719-C#1260] [General]
前陣子無聊到博客來找有沒有有趣的書，於是看了「蛤蟆先生去看心理師」這本書，想要推薦給大家。但又不想暴雷，所以以下盡量不出現劇情，並以我對憂鬱症的一些瞭解來推薦這本書。
這本書主要是在講述蛤蟆先生看了十次心理醫生的過程。並介紹了數十種的專業名詞介紹，完全展現心理諮商療程的標準範本，如果對心理諮商有疑惑的人可以從中透過故事劇情了解。
我對憂鬱症的一點點點了解:一個人開始憂鬱有時候是別人看不出來的，憂鬱症是一種情緒障礙，會讓人情緒低落、對日常活動失去興趣、感到無力，甚至會出現負面自我評價和自責等情況。在我之前看過的一些文章種有提到，憂鬱症的人幾乎不會對別人生氣(我忘了在哪看到，各位看看就好)，盡管他們會對他人憤怒但大多會隱藏這些情感，或者表現出一些被動的行為，例如撤退、回避或拒絕與他人交往等。這類話題實在不好由一個沒有專業背景的人來說，如果各位有任何疑惑或感到身心疲憊請預約學校的心理諮商，不要忽視自己的bad心情。
繼續推薦:這本書裡面蛤蟆先生憂鬱的原因是來自父母的因素，(現在國高中生更常遇到的成績壓力有時也能歸類於家長但並不是本書的原因，請自行閱讀)，我認為在閱讀時帶入腳色時的反思，是一個更加認識自己的過程，認識不同人格、個性之間的差異。
閱讀方法:一次看一章節，這類書是用來慢慢思考認識自己而不是專業科目那樣需要背誦，可以每天看一章節(不過像我就一次看完因為我等不及，更喜歡一次看完)
以上 祝各位壓力山小

**Comment by curtis1017** [comment]
這本書我覺得非常適合給大學生們或是較年輕的國高中生們，因為憂鬱症有時候會發生在學生身上，然後他們也沒有好好重視，所以會造成一些不必要的事情發生，所以我們可以透過這本書來做預防的動作。

**Comment by Chienyuuu6** [comment]
我最近也收到別人送我這本書，本身有在進行諮商，真的很能感受到「自己願意」走入諮商室，「自己願意」幫助自己瞭解自己，所帶來很大的轉變。

亞洲社會總認為心理諮商是黑暗的，但對於國外可是很流行的ㄛ！

我今天才開始看癩蛤蟆看了三個chapter，之後有收穫也會再上來發表😽😽

**Comment by Chiuhsianglin** [comment]
我姊姊大學的時候因為遇到一些雷隊友，他們當時要準備宿營，然後遇到了一些原因使得她需要去看心理醫生並靠藥物的幫助去緩解她的焦慮， 雖然我還沒有碰到過這種狀況，但我認為當自己無法解決時，真的需要一個專業的人士來幫忙，不僅可以緩解自己的壓力，會讓自己有更多的安全感。
